cdc/entry/metrics.go

```diff
@@ -18,13 +18,6 @@ import (
 )
 
 var (
-	mounterInputChanSizeGauge = prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Namespace: "ticdc",
-			Subsystem: "mounter",
-			Name:      "input_chan_size",
```

Comment:
Could you also update ticdc.json to remove it from grafana?

Reply:
Also the data-flow.dot file in /docs.

Reply:
done